source/security/incidents.html.md

```diff
@@ -0,0 +1,124 @@
+
+Incidents take priority over all other work.
+
+### Let the team know
+
+The first thing to do is let the team know an incident is happening. Post a message with a summary of the incident in progress. Use '@here' so that the whole team is aware.
+
+### Establish an incident team
+
+You should assign 2 roles to help manage the incident:
+
+1. An incident lead. This is someone who will coordinate the incident from discovery to resolution. It's recommended to pick an incident lead based on expertise, availability and knowledge of the problem.
+2. A comms lead. This person will be responsible for communicating the incident and its status to the team, across our community channels and to key stakeholders.
+
+Incidents are varied and can require different roles to resolve. You also may discover exactly which roles you need as the incident progresses. It's recommended to request roles on the team as you need them.
+
+### Start an incident report
+
+The incident team at this point should start an incident report so that they can keep track of the incident, it's impact, it's severity, a timeline of events and, once resolved, a list of actions taken out of an incident review. You can find a template for incident reports, as well as a record of all previous incidents by year, [in the INCIDENTS folder in the team drive](https://drive.google.com/drive/u/0/folders/1plXDUGFIoyzWubWNC9J_IjMiEN8eh9pO).
+
+The nature of incidents means that you may not have time to keep the report up to date as the incident is happening. If the incident team are unable to do this, you should update the incident report after it's been resolved.
+
+### Decide on severity
+
+You should decide on a severity score for your incident as soon as possible. We have 4 levels of incident severity:
+
+| Priority | Example |
+| - | - |
+| P1 | Malicious code in govuk-frontend |
+| P2 | Malicious package masquerading as govuk-frontend |
+| P3 | Design system website is down |
+| P4 | Blocking bug introduced into a govuk-frontend component |
+
+Once severity is identified, add it to the incident report. It may take a while to identify severity or the severity may change. A severity score should be decided at least before the incident review, but ideally much earlier.
+
+### Inform key stakeholders
+
+It's important to inform the DSP senior management team (SMT) about the incident as soon as possible, especially for P1 and P2 incidents. SMT can advise if further escalation is needed.
+
+Send an email to dsp-incidents@digital.cabinet-office.gov.uk with a summary of the incident to inform SMT and other key parties.
+
+If you are dealing with a [cyber security incident](https://www.ncsc.gov.uk/section/about-ncsc/incident-management) that affects the security of the platform or our users and/or their data, the CO Digital team should also be informed so that they can assist if necessary. Include report@digital.cabinet-office.gov.uk in your email to dsp-incidents to inform cyber.
```

Comment:
Is this accurate as of the DSIT transition? I'm guessing it'd still work for now, but we should probably flag this to GDS SMT and make sure we have the right details.

Reply:
I was thinking about this as I wrote it. My take is that even though we're part of DSIT technically speaking, if a security incident happened today then we'd tell CO Digital, not whoever in DSIT manages cyber issues (who would we even tell?!). I completely agree we should let SMT know and add this to the list of CO things to change.